src/store/modules/groups.js

```diff
@@ -95,10 +95,12 @@ export const actions = {
 
       commit(types.SET_ACTIVE, { groupId })
 
-      try {
-        await dispatch('fetchGroup', groupId)
-      }
-      catch (error) {
+      await dispatch('fetchGroup', groupId)
+      const hasError = Object.keys(getters['meta/status']('fetchGroup', groupId).validationErrors).length > 0
```

Comment:
I have seen this elsewhere, perhaps we need an extra property return from the getter function, I would called it `hasValidationErrors` (to distinguish it from other errors).

Reply:
Though about it, will add it!